src/ServiceControl.Audit/Infrastructure/Bootstrapper.cs

```diff
@@ -181,7 +181,7 @@ void RecordStartup(LoggingSettings loggingSettings, EndpointConfiguration endpoi
 Database Folder Size:               {ByteSize.FromBytes(folderSize).ToString("#.##", CultureInfo.InvariantCulture)}
 ServiceControl Logging Level:       {loggingSettings.LoggingLevel}
 RavenDB Logging Level:              {loggingSettings.RavenDBLogLevel}
-Transport Customization:            {settings.TransportCustomizationType},
+Transport Customization:            {settings.TransportType},
```

Comment:
Should we log transport name / display name if we can figure it out? What about the same for persistences?

Reply:
have added the persistence type to the log - same as transport is already doing